protokube/pkg/protokube/etcd_manifest.go

```diff
@@ -233,3 +238,44 @@ func buildCertificateDirectories(c *EtcdCluster) []string {
 func notEmpty(v string) bool {
 	return v != ""
 }
+
+// buildEtcdBackupManagerContainer builds a container for the standalone etcd backup manager
+func buildEtcdBackupManagerContainer(c *EtcdCluster) *v1.Container {
+	command := []string{"/etcd-backup"}
+	command = append(command, "--backup-store", c.BackupStore)
+	command = append(command, "--cluster-name", c.ClusterName)
+	command = append(command, "--data-dir", "/var/etcd/"+c.DataDirName)
+
+	// TODO: Version etcd-backup
+
+	container := v1.Container{
+		Name:    "etcd-backup",
+		Image:   "kope/etcd-backup:latest",
```

Comment:
So @chrislovecnm did a lot of work to make assets mirror-able, so that you can create your own repository.  And I do agree that we should make this replaceable, but I want to think about how we get people from today -> running backups -> running etcd manager...

Having the object (as chris suggested) will at least let us have somewhere to put the image name.  And I think asset mirroring is what you're looking for (at least I hope so!)